user-guide/toc.yml

```diff
@@ -1047,97 +1047,151 @@
-          - name: Scaling a Cassandra cluster database
-            topicUid: Scale_Cassandra_Database
-          - name: Maintaining a Cassandra cluster
-            topicUid: Maintain_Cassandra_Cluster
-          - name: Installing Cassandra Reaper
-            topicUid: Installing_Cassandra_Reaper
-          - name: Moving the Cassandra database to a different disk drive
-            topicUid: Moving_the_Cassandra_database_to_a_different_disk_drive
-          - name: Making Cassandra nodetool accessible remotely
-            topicUid: Making_Cassandra_nodetool_accessible_remotely
-          - name: Data replication and consistency configuration
-            topicUid: replication_and_consistency_configuration
-        - name: Elasticsearch database
-          topicUid: Elasticsearch_database
-          items:
-            - name: Installing Elasticsearch on a DMA via DataMiner
+            - name: Migrating existing systems to dedicated clustered storage
+              topicUid: Migrating_the_general_database_to_Cassandra
+              items:
+                - name: Migrating the SQL general database to dedicated clustered storage
+                  topicUid: Migrating_the_general_database_to_Cassandra
+                - name: Migrating the general database to a DMS Cassandra cluster
+                  topicUid: Migrating_the_general_database_to_a_DMS_Cassandra_cluster
+        - name: Configuring a dedicated indexing database
+          topicUid: Configuring_Dedicated_Indexing_Database
+          items:
+            - name: Deploying the dedicated Elasticsearch indexing database
               topicUid: Installing_Elasticsearch_via_DataMiner
-            - name: Configuring Elasticsearch backups
-              topicUid: Configuring_Elasticsearch_backups
               items:
-                - name: Taking and restoring snapshots
-                  topicUid: Configuring_Elasticsearch_backups_Windows_Linux
-                - name: Restoring backups using the Standalone Elastic Backup Tool
-                  topicUid: Configuring_Elasticsearch_backups_Windows
-            - name: Configuring indexing settings in System Center
+                - name: Installing Elasticsearch on a DMA via DataMiner
+                  topicUid: Installing_Elasticsearch_via_DataMiner
+                - name: Migrating SRM resources to Elasticsearch
+                  topicUid: Resources_migration_to_elastic
+            - name: Configuring indexing settings
               topicUid: Configuring_DataMiner_Indexing
-            - name: Configuring the Elasticsearch database
-              topicUid: Configuring_Elasticsearch_Database
+            - name: Maintaining an indexing database
+              topicUid: Configuring_Elasticsearch_indexing_database_backups
+              items:
+                - name: Configuring Elasticsearch indexing database backups
+                  topicUid: Configuring_Elasticsearch_indexing_database_backups
```

Comment:
As of now, the *Configuring Elasticsearch indexing database backups* page is the only subpage under *Maintaining an indexing database*. Perhaps we could limit this to only one main page called *Configuring Elasticsearch indexing database backups* for now and return to this structure once more pages have been added.